Pool/Models/ScoreTheme.vb

```diff
@@ -138,6 +138,9 @@ Public Class ScoreTheme
         CurrentSession.Password = String.Empty
         CurrentSession.ID = -1
         CurrentSession.IsLoggedIn = False
+        CurrentSession.Subscribed = False
+        CurrentSession.TwoFactorEnabled = False
+        CurrentSession.isTwoFactorCode = False
```

Comment:
Rename the one that says they have two-factor enabled and the one that is authenticate using two-factor.